Dockerfile

```diff
@@ -9,15 +8,18 @@ RUN mkdir -p /usr/local/share/jupyter/kernels && chmod -R 777 /usr/local/share/j
 
 ENV JULIA_PATH=/usr/local/julia
 ENV JULIA_DEPOT_PATH=/usr/local/julia
-ENV JULIA_PROJECT=/home/jupyter/.julia/environments/v1.9
+ENV JULIA_PROJECT=/home/jupyter/.julia/environments/askem
 
+RUN mkdir -p /home/jupyter/.julia/environments/askem
 COPY --from=JULIA_BASE_IMAGE /usr/local/julia /usr/local/julia
-COPY --from=JULIA_BASE_IMAGE /ASKEM-Sysimage.so /usr/local/julia/lib/ASKEM-Sysimage.so
+COPY --chown=1000:1000 --from=JULIA_BASE_IMAGE /Project.toml /home/jupyter/.julia/environments/askem
+COPY --chown=1000:1000 --from=JULIA_BASE_IMAGE /Manifest.toml /home/jupyter/.julia/environments/askem
+COPY --chown=1000:1000 --from=JULIA_BASE_IMAGE /ASKEM-Sysimage.so /home/jupyter/.julia/environments/askem
 RUN chmod -R 777 /usr/local/julia/logs
+RUN chown -R 1000:1000 /home/jupyter
+RUN chown -R 1000:1000 /usr/local/julia
```

Comment:
If we want these folders to be owned by the user, then we should use the --chown option on the copy.